xdsl/dialects/riscv.py

```diff
@@ -1564,7 +1564,9 @@ class FMVOp(RdRsOperation[FloatRegisterType, FloatRegisterType]):
 class AddOpHasCanonicalizationPatternsTrait(HasCanonicalisationPatternsTrait):
     @classmethod
     def get_canonicalization_patterns(cls) -> tuple[RewritePattern, ...]:
-        from xdsl.transforms.canonicalization_patterns.riscv import AddImmediates
+        from xdsl.transforms.canonicalization_patterns.riscv import (
```

Comment:
This is still unrelated to this PR.
I saw that you have a branch called `dalia/risc-5-canonicalization`, can you update and open a separate PR with that one when possible, while removing this change from this PR changset? thanks!